notebook/static/tree/js/notebooklist.js

```diff
@@ -1276,6 +1278,9 @@ define([
                         var upload_file = null;
                         
                         var large_reader_onload = function (event) {
+                            if (stop_singnal === true) {
+                                return;
```

Comment:
Should this also abort the file reader? https://developer.mozilla.org/en-US/docs/Web/API/FileReader/abort

Reply:
file reader is a local variable，one chunk one file reader var，we stopped before new file reader create (after last chunk uploaded)，

Reply:
should I make a commit to fix the word(singnal)?or it is already  changed after you review?

Reply:
You'll need to make a commit to fix it - my comment was just pointing it out.